react-rendering-lists/src/index.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+const pokedex = [
+  { number: '001', name: 'Bulbasaur' },
+  { number: '004', name: 'Charmander' },
+  { number: '007', name: 'Squirtle' },
+  { number: '025', name: 'Pikachu' },
+  { number: '039', name: 'Jigglypuff' }
+];
+
+const content = (
+  <ul>
+    {
+      pokedex.map(x => <li key={x.number}>{x.name}</li>)
+    }
+  </ul>
+);
+
+ReactDOM.render(
+  <ul>{content}</ul>,
```

Comment:
You don't need the extra `<ul>` here this will cause nested `<ul>` elements, you already have a `<ul>` in your `content`